src/deployment/windows.md

```diff
@@ -108,6 +108,30 @@ so, Codemagic requires
 [associating the Azure Active Directory
 and Partner Center accounts][azureadassociation].
 
+#### Github Actions CI/CD
+
+Github Actions may use the
+[Microsoft Dev Store CLI](https://learn.microsoft.com/windows/apps/publish/msstore-dev-cli/overview)
+to package applications into an MSIX and publish them to the Microsoft Store.
+The [setup-msstore-cli](https://github.com/microsoft/setup-msstore-cli)
+Github Action installs the cli so that the Action can use it for packaging
+and publishing.
+
+As packaging the MSIX utilizes the
+[`msix` pub package][msix package], the project's `pubspec.yaml`
+must contain an appropriate `msix_config` node.
+
+The application must already be present in the Microsoft Dev Center with at
+least one complete submission, and `msstore init` must be run once within
+the repository before the Action can be performed. Once complete, running
+`msix package .` and `msix publish` in a Github Action will package the
```

Comment:
Should this be `msstore package` and `msstore publish`? Please also link to the relevant docs:

1. https://learn.microsoft.com/windows/apps/publish/msstore-dev-cli/package-command
2. https://learn.microsoft.com/windows/apps/publish/msstore-dev-cli/publish-command